Src/IronPython/Modules/_fileio.cs

```diff
@@ -294,10 +308,9 @@ public FileIO(CodeContext/*!*/ context, string name, string mode = "r", bool clo
                 }
             }
 
-            public bool closefd {
-                get {
-                    return _closefd;
-                }
+            internal bool closefd {
```

Comment:
Should this be internal? Or only the setter?

Reply:
This was an intentional change (by me). I see no reason why `closefd` should be public. It is not documented anywhere in Python API, and no versions of CPython I tried (ranging from 3.4 to 3.10) have this property visible.

Reply:
Hmm, weird, I see it here. Am I looking at the wrong object?
```
>>> f = open("temp.txt")
>>> f.buffer.raw.closefd
True
```

Reply:
Oh, it seems I was looking at the wrong object. Will make it public read-only again.